EpsilonAuxBMS/Src/AuxCAN.c

```diff
@@ -0,0 +1,87 @@
+#include "cmsis_os.h"
```

Comment:
I'd consider renaming this file to something like CanSending.

Reply:
There's the receiving callback in there so would CanHandler or something be suitable?

Reply:
Yeah!